### PR TITLE
Added instructions

### PR DESCRIPTION
**Pull request by melissahugs**
Instructions on how to git git.

### CODE REVIEW

File: mah-melizza.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+print('Hello world')
+
+# git status
+# git add .
+# git git commit -m 'update the stuffs'
+# git push
+
+input ("How do dis")
```
Comment (gnalvesteffer) [inline mah-melizza.py]: You misspelled "this".

File: mah-melizza.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+print('Hello world')
```
Comment (AnneOReilly) [inline mah-melizza.py:1]: You are a spell

**Review by AnneOReilly — APPROVED** [review]

**Review by gnalvesteffer — CHANGES_REQUESTED** [review]
Looks pretty good, but you have a spelling error.

File: mah-melizza.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+# git status
+# git add .
+# git git commit -m 'update the stuffs'
+# git push
```
Comment (ShipDandy) [inline mah-melizza.py:6]: i really like pizza, include pizza.

Comment (gnalvesteffer) [reply mah-melizza.py:6]: Agreed, pizza should be added.

File: mah-melizza.py (hunk 2 above)
Comment (gnalvesteffer) [reply mah-melizza.py:1]: ❓

File: mah-melizza.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+input ("How do this, okay??")
```
Comment (gnalvesteffer) [inline mah-melizza.py:8]: Thanks for correcting that spelling error!

**Review by gnalvesteffer — APPROVED** [review]
:+1:

**Pull request merged**